### DISCUSSION TITLE
disable page scrolling on resume

### DISCUSSION THREAD

**Discussion by EmilioPanti** [jcubic/jquery.terminal#713] [Q&A] [answered]
Hi,
when I call the resume function on the terminal, which was previously paused, the page is scrolled to the page height of the terminal. It happens on both chrome and firefox.
Is there any way to prevent this behavior?

**Comment by jcubic** [comment, marked as answer]
It seems not, but I can make it use return value from `onResume` event:
```javascript
onResume() {
   return false;
}
```
return value from that event is not used.

**Reply by jcubic** [reply]
Added as feature #800

**Reply by jcubic** [reply]
@[EmilioPanti](https://github.com/EmilioPanti) The change was added to the devel branch. It will be released in the next version.
